ricq-core/src/command/online_push/decoder.rs

```diff
@@ -12,7 +12,7 @@ impl super::super::super::Engine {
     // 解析群消息分片 长消息需要合并
     // OnlinePush.PbPushGroupMsg
     pub fn decode_group_message_packet(&self, payload: Bytes) -> RQResult<GroupMessagePart> {
-        let message = pb::msg::PushMessagePacket::decode(payload)?;
+        let message = pb::msg::PushMessagePacket::decode(&*payload)?;
```

Comment:
Use Deref on `bytes::Bytes` will produce the `&[u8]`, and avoid much generics monomorphization.

The `prost`'s `decode` function includes lots of code for parsing and serialization, pass `Bytes` directly caused 68 KiB binary size increasement on my app.